.github/dependabot.yml

```diff
@@ -28,6 +28,20 @@ updates:
       # arrow is bumped manually
       - dependency-name: "arrow*"
         update-types: ["version-update:semver-major"]
+  - package-ecosystem: cargo
```

Comment:
nice, but we mostly take the dependencies from datafusion?

Reply:
Yeah I don't know if the check for datafusion is really needed

For arrow, it looks like cli does have an explicit dependency on it though:

https://github.com/apache/arrow-datafusion/blob/4ad4f90d86c57226a4e0fb1f79dfaaf0d404c273/datafusion-cli/Cargo.toml#L33